pyenv-lazy.plugin.zsh

```diff
@@ -1,5 +1,17 @@
-# Try to find pyenv, if it's not on the path
 export PYENV_ROOT="${PYENV_ROOT:=${HOME}/.pyenv}"
+
+# Try to install pyenv, if $PYENV_ROOT is empty
+function _zsh_pyenv_install() {
+    echo "Installing pyenv..."
+    git clone "https://github.com/pyenv/pyenv.git" "${PYENV_ROOT}"
+    if [[ -n "${ZSH_PYENV_LAZY_VIRTUALENV}" ]]; then
+        echo "Installing pyenv-virtualenv..."
+        git clone "https://github.com/pyenv/pyenv-virtualenv.git" "${PYENV_ROOT}/plugins/pyenv-virtualenv"
+    fi
```

Comment:
I've installed using `brew`, since I'm on mac. Installing via git has the benefit of being independent of package managers, but has the downside of being more difficult to update, for example when new Python versions are installed.

Any thoughts on this?

Reply:
Nice! But only for the plugin, not for pyenv itself. 😭

If I had the time I'd fix that myself but sadly I don't at the moment.